stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/cookbook/ui/ChatView.kt

```diff
@@ -68,7 +68,7 @@ class ChatView : Fragment() {
 
         // Let the message input know when we are editing a message
         messageListView.setMessageEditHandler { message ->
-            messageInputViewModel.editMessage.postValue(message)
+            messageInputViewModel.postMessageToEdit(message)
```

Comment:
You can use `messageListView.setMessageEditHandler(messageInputViewModel::postMessageToEdit)`